pytorch_lightning/utilities/cli.py

```diff
@@ -277,8 +277,7 @@ def __init__(
             seed_everything(seed, workers=True)
 
         self.before_instantiate_classes()
-        self.instantiate_classes(self.subcommand)
-        self.add_configure_optimizers_method_to_model(self.subcommand)
```

Comment:
Move this inside `instantiate_classes` so users do not need to call them separately

Reply:
Good !